### ISSUE TITLE
[Suggestion] Remove Tag syntax from UK domains

### ISSUE THREAD

**Issue by zaerl** [Automattic/wp-calypso#82454]
See http://calypso.localhost:3000/site-profiler?domain=google.co.uk

UK domains output registrar's names as `Markmonitor Inc. [Tag = MARKMONITOR]`. Removing the `[Tag]` suffix can help clean up the output.

**Comment by bogiii** [comment]
This is an exceptional case (we can handle it), but millions of similar cases must be around, and if you asked me, I would leave it as it is.

What do others have to say?
@vishnugopal @ouikhuan

**Comment by zaerl** [comment]
> This is an exceptional case (we can handle it), but millions of similar cases must be around, and if you asked me, I would leave it as it is.

This is not. All `.co.uk` domains I tried has the same output. It only change the value of the "Tag".

1. https://wpcalypso.wordpress.com/site-profiler?domain=amazon.co.uk
2. https://wpcalypso.wordpress.com/site-profiler?domain=google.co.uk
3. https://wpcalypso.wordpress.com/site-profiler?domain=bbc.co.uk
4. https://wpcalypso.wordpress.com/site-profiler?domain=ebay.co.uk
5. https://wpcalypso.wordpress.com/site-profiler?domain=service.co.uk
6. https://wpcalypso.wordpress.com/site-profiler?domain=mirror.co.uk
7. Etc.

**Comment by ouikhuan** [comment]
> This is an exceptional case (we can handle it), but millions of similar cases must be around, and if you asked me, I would leave it as it is.
> 
> What do others have to say? @vishnugopal @ouikhuan

This will definitely be a nice thing to handle but I feel this is more like a bonus and not the first priority in i1 so I'll probably place it in the next iteration for now unless we have more bandwidth to handle this.

**Comment by vishnugopal** [comment]
Agreed on putting this in i2. This is a small regex search replace fix and we should be aware that there may be more of these.
